tests/auth-react/django3x/mysite/utils.py

```diff
@@ -2,6 +2,8 @@
 from typing import Any, Dict, List, Optional, Union
 
 from dotenv import load_dotenv
+from typing_extensions import Literal
```

Comment:
Reset unrelated changes in this file

Reply:
`dashboard/__iniy__.py` doesn't globally import `dashboard/recipe.py` so these imports change